meson.build

```diff
@@ -0,0 +1,21 @@
+project('minrx', ['c', 'cpp'],
+        version : '0.0.1',
+        default_options : ['warning_level=3', 'cpp_std=c++20'])
+
+libminrx_inc = include_directories('.')
+libminrx_lib = shared_library('minrx',
```

Comment:
I am new to meson, but it seems just `library()` would be preferred. It defaults to a shared library but you can build the static on if desired by adding `-Ddefault_library=static` to meson.